### DISCUSSION TITLE
About Windows default (balanced) power plan

### DISCUSSION THREAD

**Discussion by Konnos998** [seerge/g-helper#1724] [General]
You say that it's strongly recommended to run app with Windows default "balanced" power plan. I want to change some settings inside the Windows power plan (such as the processor performance, sleep times, intel hd graphics -> best performance. etc...and stuff like that). Is it okay to do so or there'll be problems, for example G-Helper will get confused and may override values when I change the G-Helper modes (silent, balanced, turbo)?

**Comment by seerge** [comment]
App doesn't touch any of the power plan settings, except CPU boost (that it offers a shortcut to)

**Reply by Konnos998** [reply]
I asked the question because 1-2 days ago, sometimes my mouse stucks a bit (something like a lag) and I was wondering if it's due to the power plan settings...But I don't know what it is. I want to see if it's G-Helper or something else.
Do you suggest me to download the latest pre-release you've uploaded (v0.136), because now I have v0.134?

**Reply by seerge** [reply]
I don't think mouse has something to do with windows power plans :) you can always reset power plan to defaults and see if make a difference

G-Helper is not a driver, it's just an app :) it doesn't control your mouse anyhow

**Reply by Konnos998** [reply]
Thanks a lot! Maybe I found what was it (usb selective suspension was enabled and that might caused this micro-freeze for the mouse). Thanks a lot though!
